pr-data.csv

```diff
@@ -166,7 +166,7 @@ https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1
 https://github.com/Adyen/adyen-java-api-library,6fb5cd049b57a22d2ec4465d204c15f1c90dd325,.,com.adyen.service.CheckoutTest.testPaymentsRequestWithXidAndCavv,ID,Accepted,https://github.com/Adyen/adyen-java-api-library/pull/681,
 https://github.com/akamai/netty,cee60304dea4eeb779aa60af08332b1a965de984,testsuite,io.netty.testsuite.transport.socket.SocketSslEchoTest.testSslEcho,ID,,,
 https://github.com/alibaba/asyncload,9891e7086bff81ebdd4991435d21b549e48e874d,.,com.alibaba.asyncload.AsyncLoadThreadLocalTest.testInheritableThreadLocalRunnerSet,OD-Vic,,,
-https://github.com/alibaba/druid,ad05a9293b6be2d5562807698b215b673db0417a,pool,com.alibaba.druid.bvt.pool.DruidDataSourceTest_exceptionSorter.test_event_error,ID,,,
+https://github.com/alibaba/druid,ad05a9293b6be2d5562807698b215b673db0417a,core,com.alibaba.druid.bvt.pool.DruidDataSourceTest_exceptionSorter.test_event_error,ID,MovedOrRenamed,,https://github.com/alibaba/druid/commit/1e93276be86bcf5f8cf114b08689e37f27da8721
```

Comment:
Please don't change the module in this row (keep `pool`) and add a new row with the module as `core` and empty status and note